redeem/gcodes/M2x.py

```diff
@@ -288,51 +306,53 @@ class M24(GCodeCommand):
     def process_gcode(self, fn, g):
 
         with open(fn, 'r') as gcode_file:
-            logging.info("M23: file open: '{}".format(fn))
+            logging.info("M24: file open: '{}'".format(fn))
 
             count = sum(1 for line in gcode_file)
-            logging.info("M23: line count: '{}".format(count))
+            logging.info("M24: line count: '{}'".format(count))
             self.printer.sd_card_manager.current_lock.acquire()
             self.printer.sd_card_manager.current_line_count = 0
             self.printer.sd_card_manager.current_file_count = count
+            self.printer.sd_card_manager.current_byte_count = 0
             self.printer.sd_card_manager.current_lock.release()
 
             gcode_file.seek(0)
 
             for line in gcode_file:
                 line = line.strip()
-                logging.info("line is '{}'".format(line))
+                #logging.info("line is '{}'".format(line))
```

Comment:
all of these can be removed too if they aren't needed. i was probably too verbose in adding logging.